### DISCUSSION TITLE
Use Oxalis for SMP Lookup

### DISCUSSION THREAD

**Discussion by francescodiperna** [OxalisCommunity/Oxalis-AS4#170] [General]
Hi community,
we used Oxalis to pass the Testbed Openpeppol but now, without documentation, we have some troubles.
Can I ask you how to use oxalis for lookup to an SMP to recover the Service Metadata envelope?

Kind regards
Di Perna Francesco

**Comment by aaron-kumar** [comment]
Refer this : https://github.com/OxalisCommunity/vefa-peppol/tree/master/peppol-lookup
